_sass/icons.scss

```diff
@@ -0,0 +1,56 @@
+@font-face {
+  font-family: 'fontello';
+  src: url('/assets/fonts/fontello.eot?46807042');
+  src: url('/assets/fonts/fontello.eot?46807042#iefix') format('embedded-opentype'),
+       url('/assets/fonts/fontello.woff?46807042') format('woff'),
+       url('/assets/fonts/fontello.ttf?46807042') format('truetype'),
+       url('/assets/fonts/fontello.svg?46807042#fontello') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+ 
+ [class^="icon-"]:before, [class*=" icon-"]:before {
```

Comment:
Line should be indented 0 spaces, but was indented 1 spaces<br>Rule set contains (15/10) properties<br>Each selector in a comma sequence should be on its own single line